components/TextFields/src/MDCTextInputControllerBase.m

```diff
@@ -41,6 +41,7 @@
     MDCTextInputControllerBaseDefaultFloatingPlaceholderDownAnimationDuration = (CGFloat)0.266666;
 static const NSTimeInterval
     MDCTextInputControllerBaseDefaultFloatingPlaceholderUpAnimationDuration = (CGFloat)0.3;
+static const NSTimeInterval kDefaultErrorAnnouncementDelay = (CGFloat)0.500;
```

Comment:
_Question:_ Where is this value coming from?

Reply:
I just made it up. It seems sufficient to vocalize most text input. It slightly mitigates the error announcement interruption effect.